route/v1/types.go

```diff
@@ -145,8 +146,155 @@ type RouteSpec struct {
 	// +kubebuilder:validation:Enum=None;Subdomain;""
 	// +kubebuilder:default=None
 	WildcardPolicy WildcardPolicyType `json:"wildcardPolicy,omitempty" protobuf:"bytes,7,opt,name=wildcardPolicy"`
+
+	// httpHeaders defines policy for HTTP headers.
+	//
+	// If this field is empty, the default values are used.
+	//
```

Comment:
@JoelSpeed, do we need this statement? It doesn't make much sense in the context of the Route API; I believe it was copied from the IngressController API, where `spec.httpHeaders` has other fields besides `actions` that do have meaningful defaults.

Reply:
@Miciah I will remove it

Reply:
SGTM